### ISSUE TITLE
Querying logs fails when JSON set to false

### ISSUE THREAD

**Issue by Grayda** [winstonjs/winston-daily-rotate-file#149]
I've got this piece of code, running winston 2.4.2 and winston-daily-rotate-file 3.1.4 on node 8.7.0 (Windows 10, x64):

```
var winston = require("winston")
var winston_logrotate = require("winston-daily-rotate-file")

winston.add(winston.transports.DailyRotateFile, {
  filename: "sample.log"
})

winston.log("info", "This is a sample log entry")

winston.query({}, function(err, results) {
  if (err) {
    throw err;
  }

  console.dir(results.dailyRotateFile)
})
```

When I run it, I get this error:

```
info: This is a sample log entry
SyntaxError: Unexpected number in JSON at position 4
    at JSON.parse (<anonymous>)
    at add (C:\Users\graydav1\Documents\GitHub\bellboy\node_modules\winston-daily-rotate-file\daily-rotate-file.js:226:32)
    at ReadStream.<anonymous> (C:\Users\graydav1\Documents\GitHub\bellboy\node_modules\winston-daily-rotate-file\daily-rotate-file.js:200:21)
    at emitOne (events.js:115:13)
    at ReadStream.emit (events.js:210:7)
    at addChunk (_stream_readable.js:264:12)
    at readableAddChunk (_stream_readable.js:247:13)
    at ReadStream.Readable.push (_stream_readable.js:209:10)
    at onread (fs.js:2059:12)
    at FSReqWrap.wrapper [as oncomplete] (fs.js:665:17)
```

It looks like the JSON.parse error isn't being picked up by the try / catch block.

I can work around this by setting the `json` option to true, but this should work without JSON.

**Comment by mattberther** [comment]
@Grayda querying the logs will not be possible without the `json` option being set to true. I'll add a check to the query method to enforce the required `json` setting.

**Comment by Grayda** [comment]
Makes sense, but the docs don't mention that, so I assumed it could be done.

Thanks :)

**Comment by mattberther** [comment]
Resolved with #152 and published to npm as `winston-daily-rotate-file@3.2.0`.
